src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -525,14 +526,17 @@ public String getQuestionAdditionalInfoHtml(int questionNumber, String additiona
     
     @Override
     public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> responses,
-            FeedbackQuestionAttributes question,
-            String studentEmail,
-            FeedbackSessionResultsBundle bundle,
-            String view) {
+            FeedbackQuestionAttributes question, String studentEmail,
+            FeedbackSessionResultsBundle bundle, String view) {
+
+        List<FeedbackResponseAttributes> responsesForStatistics =
+                filterResponsesForStatistics(responses, question, studentEmail, bundle, view);
+
+        FeedbackRubricQuestionDetails feedbackQuestionDetails =
```

Comment:
Again, unless it is necessary for this issue, do avoid making unrelated edits. Open a separate issue if you feel that the naming can be improved